crates/metadata/lib.rs

```diff
@@ -71,6 +71,7 @@ pub const DEFAULT_TARGETS: &[&str] = &[
 #[non_exhaustive]
 pub enum MetadataError {
     /// The error returned when the manifest could not be read.
+    #[allow(clippy::upper_case_acronyms)]
```

Comment:
This change doesn't belong in this PR, it was already fixed in master.

Reply:
:+1: